### ISSUE TITLE
2.2.1 is not tagged in git and versions >2.2.0 are not listed on your homepage

### ISSUE THREAD

**Issue by stefanruijsenaars** [scottjehl/picturefill#474]
Just to let you know, in Drupal 8 we use version 2.2.1 of picturefill, but there is no git tag for this release.

Could you tag this? Or alternatively could you explain how 2.3.0 differs from 2.2.1? 

Lastly just to clarify, is there a reason 2.2.1/2.3.0 are not listed on your homepage (http://scottjehl.github.com/picturefill/)

**Comment by jegtnes** [comment]
Hey @stefanruijsenaars, thanks for reporting this!

I think 2.2.1 was short-lived as 2.3.0 was scheduled for an imminent release. I've created two pull requests to update the site to reflect that 2.3.0 is the recommended stable version.

2.3.0 fixes a range of bugs and has some performance improvements, as well as adding support for newer image formats, intrinsic image dimensions, and commonJS exports. More details here: https://github.com/scottjehl/picturefill/releases

Basically, we heartily recommend you upgrade to 2.3.0. It's the bee's knees. 🐝

**Comment by stefanruijsenaars** [comment]
Thanks!
